lib/routes/api/service-providers.js

```diff
@@ -128,6 +128,12 @@ router.get('/:sid/VoipCarriers', async(req, res) => {
     await validateRetrieve(req);
     const service_provider_sid = parseServiceProviderSid(req);
     const results = await VoipCarrier.retrieveAllForSP(service_provider_sid);
+    console.log(results);
```

Comment:
please use logger not console.log

Reply:
removed completely, this was an oversight.